Classes/Exception/MissingParameterException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace WebVision\Deepltranslate\Core\Exception;
+
+final class MissingParameterException extends \Exception
```

Comment:
Please add soe coment which this exception means when thrown. Eventually point to place(s) which throws this.